test/echo-server.c

```diff
@@ -96,7 +96,7 @@ static void after_read(uv_stream_t* handle,
    * Scan for the letter Q which signals that we should quit the server.
    * If we get QS it means close the stream.
    */
-  if (!server_closed) {
+  if (0 && !server_closed) {
```

Comment:
Is this `0` intentional?

Reply:
Yes, I can't have the echo-server exiting just because it saw the bytes `QS`. This would only used by the performance benchmarks, and I can't find a way to run those.

Reply:
> I can't have the echo-server exiting just because it saw the bytes QS

Why not?  Are you feeding it random data?

Reply:
Yes. In the initial CI run, it seemed like some of the bots couldn't allocate 2+ gigabytes of all zeros, so I switched to using random data.